components/SCDDetect.coffee

```diff
@@ -8,25 +8,34 @@ exec = require('child_process').exec
 # @name SCDDetect
 
 compute = (canvas, cascade, callback) ->
-  # Get canvas
   ctx = canvas.getContext '2d'
```

Comment:
These 3 lines are unused? the `.getImageData` call also does a non-trivial amount of computation I'd guess